simtools/simtel/simtel_io_histogram.py

```diff
@@ -78,6 +87,11 @@ def __init__(self, histogram_file, area_from_distribution=False):
         self.radius_axis = None
         self.area_from_distribution = area_from_distribution
 
+        self.view_cone = view_cone
```

Comment:
I would find this clearer with `self.view_cone = self._set_view_cone(viewcone)`: in the current implementation, the first assignment is immediately overwritten by the follow up statement.

Same comment for the energy range in lines 93/94.

Reply:
I understand your point. I improved it now.